src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const avarageTime = 3;
```

Comment:
The variable name 'avarageTime' has a typo. It should be 'averageTime'. Also, it's better to name it more descriptive, for example 'shortTerm'.